migration/20160614-1-remove-licensepools-from-series-lanes.sql

```diff
@@ -0,0 +1 @@
+UPDATE cachedfeeds set license_pool_id=NULL where type='series';
```

Comment:
Rename this file to 20160617-... to avoid (my) confusion). I just run a bunch of these migration scripts, including a 20160615, and the 17th will be the earliest we'll be able to run this script for real.

Eventually these will be numbered with version numbers instead of dates. That won't eliminate the need to rename files occasionally but it should make it less common.